<?xml version="1.0"?>
<launch>
	<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">		

                <param name="video_device" type="string" value="/dev/video0"/>
                <param name="image_width" type="int" value="640" />
                <param name="image_height" type="int" value="480" />
                <param name="camera_frame_id" type="string" value="usb_cam" />
                <param name="framerate" type="int" value="30" />
                <param name="autofocus" type="bool" value="true" />
		
	</node>

        <node name="exercise03" pkg="oberstufenlabor_mechatronik_vision" type="exercise03" output="screen">
                <param name="image_topic" type="string" value="usb_cam/image_raw"/>
        </node>
                
</launch>

<!-- Launch for Intel Cam
<launch>
	<node name="intel_cam" pkg="imes_intel_cam" type="intel_cam" output="screen">		

		<param name="ColorFrameRate"   type="int"    value="25"  />
		<param name="DepthFrameRate"   type="string" value="25" />
		<param name="ColorFrameFormat" type="string" value="640x360" />
		<param name="DepthFrameFormat" type="string" value="320x240" />

                <param name="enable/ColorMap"          type="bool" value="true" />
                <param name="enable/PointCloud"        type="bool" value="false" />
                <param name="enable/Tf"                type="bool" value="false" />
                <param name="enable/CameraParameters"  type="bool" value="false" />

                <param name="TfFrequency"       type="double" value="5"/>
                <param name="CamParamFrequency" type="double" value="5"/>
		
	</node>

        <node name="exercise03" pkg="oberstufenlabor_mechatronik_vision" type="exercise03" output="screen">
                <param name="image_topic" type="string" value="usb_cam/image_raw"/>
        </node>
                
</launch>
-->
